### PR TITLE
Upgrade to Micronaut 4.3.8

### PR DESCRIPTION
**Pull request by mraible**
https://micronaut.io/2024/04/10/micronaut-framework-4-3-8-released/

### CODE REVIEW

File: generators/constants.cjs [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 module.exports = {
   JAVA_VERSION: 17,
-  GRADLE_VERSION: '8.6',
+  GRADLE_VERSION: '8.7',
```
Comment (mshima) [inline generators/constants.cjs:22]: Inherited now.
```suggestion
```

**Comment by mshima** [comment]
I have a partially prepared followup to https://github.com/jhipster/generator-jhipster-micronaut/pull/334 that adds the micronaut to dependabot.

**Comment by mraible** [comment]
Please proceed now that #334 is merged.

**Comment by DanielFran** [comment]
Superseded by https://github.com/jhipster/generator-jhipster-micronaut/pull/337

**Comment by mraible** [comment]
#337 does not upgrade Micronaut to 4.3.8. Maybe a dependabot PR will do it soon?

**Comment by DanielFran** [comment]
> #337 does not upgrade Micronaut to 4.3.8. Maybe a dependabot PR will do it soon?

Tomorrow morning at 8:00
image

**Pull request closed without merging**